src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -280,6 +285,119 @@ public void shutdown(ExecutorDriver d) {
     log.info("Executor asked to shutdown");
   }
 
+  protected void runHealthChecks(ExecutorDriver driver, Task task) {
+    log.info("Performing health check for task: " + task.taskInfo.getTaskId().getValue());
+    
+    String nodeName = null;
+    String healthCheckCmd = "netstat -plnat | grep ";
```

Comment:
I'm thinking this should happen in java:

public static boolean serverListening(String host, int port) {
    Socket s = null;
    try {
        s = new Socket(host, port);
        return true;
    } catch (Exception e) {
        return false;
    } finally {
        if(s != null)
            try {s.close();}
            catch(Exception e){}
    }
}